app/deliver_grant_funding/routes/reports.py

```diff
@@ -111,15 +112,25 @@ def change_report_name(grant_id: UUID, report_id: UUID) -> ResponseReturnValue:
     )
 
 
-@deliver_grant_funding_blueprint.route("/grant/<uuid:grant_id>/report/<uuid:report_id>", methods=["GET"])
+@deliver_grant_funding_blueprint.route("/grant/<uuid:grant_id>/report/<uuid:report_id>", methods=["GET", "POST"])
 @has_grant_role(RoleEnum.MEMBER)
+@auto_commit_after_request
 def list_report_tasks(grant_id: UUID, report_id: UUID) -> ResponseReturnValue:
     report = get_collection(report_id, grant_id=grant_id, type_=CollectionType.MONITORING_REPORT, with_full_schema=True)
+    form = GenericSubmitForm()
 
     if report.has_non_default_sections:
         raise RuntimeError(f"Report {report_id} has non-default sections - `add_task` needs updating to handle this.")
 
-    return render_template("deliver_grant_funding/reports/list_report_tasks.html", grant=report.grant, report=report)
+    if form.validate_on_submit() and form.submit.data:
+        return start_testing_submission(collection=report)
```

Comment:
Probably a test in `TestListReportTasks` for POSTing the form - can just check that it does the redirect to the expected place?